config-linux.md

```diff
@@ -494,6 +494,21 @@ You MUST specify at least one of the `hcaHandles` or `hcaObjects` in a given ent
 }
 ```
 
+## <a name="configLinuxUnified" />Unified
+
+**`unified`** (object, OPTIONAL) allows cgroup v2 parameters to be modified at runtime for the container.
+
+Each key in the map refers to a file in the cgroup unified hierarchy.  The OCI runtime must ensure that the needed cgroup controllers are enabled for the cgroup.
```

Comment:
```markdown
Restrictions:
* A key MUST NOT contain "/".
* A key MUST NOT start with "cgroup.".
```

Reply:
I agree on the first one, but why the second?  Wouldn't be useful to set `cgroup.type`?

Reply:
Usecase?

Reply:
just have the freedom of doing it :-)

Reply:
:-)

At least we should deny setting `cgroup.subtree_control`? And also `cgroup.procs`, `cgroup.threads`, `cgroup.freeze` probably.

Reply:
There are at least two: `cgroup.max.descendants` and `cgroup.max.depth` that makes sense to override.

Should the block list be specified in the OCI specs or is it an implementation detail and left to the OCI runtime?  @cyphar what do you think?

Reply:
ping